plugins/org.eclipse.fordiac.ide.typemanagement/src/org/eclipse/fordiac/ide/typemanagement/wizards/NewFBTypeWizardPage.java

```diff
@@ -228,6 +235,21 @@ private boolean isDuplicate() {
 		return types.containsKey(getFullTypeName());
 	}
 
+	private boolean fileExists() {
+		final String fileName = getFileName().toLowerCase();
+		try {
+			for (final IResource member : ResourcesPlugin.getWorkspace().getRoot().getFolder(getContainerFullPath())
+					.members()) {
+				if (fileName.equals(member.getName().toLowerCase())) {
```

Comment:
would an equalsIgnoreCase not be nicer to read?

Reply:
it would - changed it